src/main/resources/static/css/stylesheet.css

```diff
@@ -57,6 +61,24 @@ nav ul li a {
   height: auto;
 }
 
+#cart_pic {
+  position: relative;
+  width: 30px;
+  height: 30px;
+  text-align: right;
+  float: right;
+  padding: 10px;
+}
+
+#user_pic {
+  position: relative;
+  width: 30px;
+  height: 30px;
+  text-align: right;
+  float: right;
+  padding: 10px;
+}
+
```

Comment:
I would recommend a single css class called something like "header-pic" that you could use for both of these pictures?